storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/JdbcHistoricalRetriever.java

```diff
@@ -326,8 +377,8 @@ String runBatchQuery(
       ResultSet rs = statement.executeQuery(timestampLimitSqlQuery);
 
       while (rs.next()) {
-        Timestamp min_ts = rs.getTimestamp("min"); // Get minimum timestamp
-        Timestamp max_ts = rs.getTimestamp("max"); // Get maximum timestamp
+        Timestamp min_ts = rs.getTimestamp(2); // Get minimum timestamp
```

Comment:
Should this be reverted?

Reply:
the current implementation is correct. In the templater, "SELECT max(event_timestamp) as max, min(event_timestamp) as min from %s;". So Max is the first column and min is the second column. But I can clean up the order in order to make it easy to read.